examples/draft-0-11/table/src/components/RichTextExample.js

```diff
@@ -1,72 +1,84 @@
 import React from 'react';
 import {Editor, EditorState} from 'draft-js';
 import {Map} from 'immutable';
-import BlockComponent from "./BlockComponent";
-import ModalTable from "./Table/ModalTable";
-import createTable from "./Table/modifiers/createTable";
+import BlockComponent from './BlockComponent';
+import ModalTable from './Table/ModalTable';
+import createTable from './Table/modifiers/createTable';
 import './RichTextEditor.css';
 import './Table/Table.css';
 
 function MyEditor() {
-	const [editorState, setEditorState] = React.useState(
-		EditorState.createEmpty()
-	);
-	const [tableEdits, setTableEdits] = React.useState(Map());
+  const [editorState, setEditorState] = React.useState(
+    EditorState.createEmpty(),
+  );
+  const [tableEdits, setTableEdits] = React.useState(Map());
 
-	const editor = React.useRef(null);
+  const editor = React.useRef(null);
 
-	const blockRenderer = (block) => {
-		if (block.getType() === 'atomic') {
-			return {
-				component: BlockComponent,
-				editable: false,
-				props: {
-					onStartEdit: (blockKey) => {
-						setTableEdits(tableEdits.set(blockKey, true))
-					},
-					onFinishEdit: (blockKey) => {
-						setTableEdits(tableEdits.remove(blockKey))
-					},
-				},
-			}
-		}
-		return null
-	}
+  const blockRenderer = block => {
+    if (block.getType() === 'atomic') {
+      return {
+        component: BlockComponent,
+        editable: false,
+        props: {
+          onStartEdit: blockKey => {
+            setTableEdits(tableEdits.set(blockKey, true));
+          },
+          onFinishEdit: blockKey => {
+            setTableEdits(tableEdits.remove(blockKey));
```

Comment:
On Finish Edit the editor still stays on read only mode.  Do we have any other workaround?

Reply:
The feature is just experimental, 
To be honest, it is just a toy, and
now I have no time to develop it in this month or longer, 
you can try the website： https://zhihu.com/, it also uses draft-js and has implemented table feature.

Reply:
Anyway thanks. Your PR saved me a lot of time.